packages/wasm-dpp/src/state_transition/mod.rs

```diff
@@ -69,6 +72,57 @@ impl StateTransitionExecutionContextWasm {
     pub fn disable_dry_run(&self) {
         self.0.disable_dry_run();
     }
+
+    #[wasm_bindgen(js_name=isDryRun)]
+    pub fn is_dry_run(&self) -> bool {
+        self.0.is_dry_run()
+    }
+
+    #[wasm_bindgen(js_name=addOperation)]
+    pub fn add_operation(&self, operation: JsValue) -> Result<(), JsValue> {
+        let operation = create_operation_from_wasm_instance(&operation)?;
+        self.0.add_operation(operation);
+        Ok(())
+    }
+
+    #[wasm_bindgen(js_name=getDryOperations)]
+    pub fn get_dry_operations(&self) -> Vec<JsValue> {
+        self.0
+            .get_dry_operations()
+            .iter()
+            .map(|operation| match operation {
+                Operation::PreCalculated(operation) => {
+                    PreCalculatedOperationWasm::from(operation.to_owned()).into()
+                }
+                Operation::Read(operation) => ReadOperationWasm::from(operation.to_owned()).into(),
+                Operation::SignatureVerification(operation) => {
+                    SignatureVerificationOperationWasm::from(operation.to_owned()).into()
+                }
+            })
+            .collect()
+    }
+
+    #[wasm_bindgen(js_name=getOperations)]
+    pub fn get_operation(&self) -> Vec<JsValue> {
```

Comment:
Whopsie, a typo! Didn't notice that when I was reviewing the PR from which this code comes from. Not thatit really matters